Objects/dictobject.c

```diff
@@ -2804,18 +2919,22 @@ dict_clear(PyDictObject *mp)
 }
 
 static PyObject *
-dict_pop(PyDictObject *mp, PyObject *args)
+dict_pop(PyDictObject *mp, PyObject *args, PyObject *kwargs)
```

Comment:
Open a separate issue for this feature.

Adding support of keyword arguments slows down arguments parsing.